pkgs/build-support/kernel/make-initrd.nix

```diff
@@ -31,7 +34,8 @@ in stdenvNoCC.mkDerivation rec {
   makeUInitrd = stdenvNoCC.hostPlatform.platform.kernelTarget == "uImage";
 
   nativeBuildInputs = [ perl cpio ]
-    ++ stdenvNoCC.lib.optional makeUInitrd ubootTools;
+    ++ stdenvNoCC.lib.optional makeUInitrd ubootTools
```

Comment:
nit: we're taking lib as an arg, so we don't need to use `stdenvNoCC` to get to it :) though since this was already like this I don't think it needs to be changed in the context of this PR.